apps/heft/src/cli/HeftActionRunner.ts

```diff
@@ -291,9 +309,13 @@ export class HeftActionRunner {
 
     initializeHeft(this._heftConfiguration, terminal, this.parameterManager.defaultParameters.verbose);
 
-    const operations: ReadonlySet<Operation> = this._generateOperations();
+    const operations: ReadonlySet<Operation<IHeftTaskOperationMetadata, IHeftPhaseOperationMetadata>> =
+      this._generateOperations();
 
-    const executionManager: OperationExecutionManager = new OperationExecutionManager(operations);
+    const executionManager: OperationExecutionManager<
+      IHeftTaskOperationMetadata,
+      IHeftPhaseOperationMetadata
+    > = new OperationExecutionManager<IHeftTaskOperationMetadata, IHeftPhaseOperationMetadata>(operations);
```

Comment:
```suggestion
    > = new OperationExecutionManager(operations);
```
We don't usually specify the generic twice